robot.py

```diff
@@ -245,7 +238,7 @@ def initSwervometer(self, config):
         return swervometer
 
     def initVision(self, config):
-        vision = Vision(NetworkTables.getTable('limelight'),
+        vision = Vision(NetworkTables.getTable('limelight-g'),
```

Comment:
It'd suggest putting limelight-g and limelight-note into robotconfig so they don't appear as magic values elsewhere.